target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
```

Comment:
Please add a license.

Reply:
Sure, NP - added. Will push latest updates in just a bit.